Jenkinsfile

```diff
@@ -2,7 +2,8 @@ pipeline {
     agent {
         dockerfile {
             /* There are resolution issues on the Jenkins server for pelux.io */
-            args "--add-host=pelux.io:${env.JENKINS_IP}"
+            // This is IP is hardcoded until the domain is updated to "New Jenkins"
+            args "--add-host=pelux.io:185.89.242.33"
```

Comment:
This as always is quite dangerous, I hope you put it some measures in so this will be not forgotten to be removed as soon as the domain is updated?

Reply:
I don't want to keep this solution, but this was the way I could easily fix it while testing.